### DISCUSSION TITLE
Suggestion : Flutter (Dart ) Code support

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [uxbox/uxbox#459] [Ideas]
**I am referring  to something like this plugin from adobe:**
https://blog.adobe.com/en/publish/2020/09/10/xd-flutter-plugin-now-available.html#gs.q58ukl

As far as i have searched there is no **OpenSource** software that does this, so if somewhere along the road you can achieve this i believe would be huge!!
Thank you all so much for your work!!
Is this in closed Alpha ? Cause id love to try it!

**Comment by alt-tosc** [comment]
Yes indeed, as you said, I also believe that it would be huge! :)

I cannot resist mentioning the opensource project named [Fidget](https://github.com/treeform/fidget). I will not do the comparison but I see some similarities. Fidget leverages Figma and it uses a great programming language named [Nim](https://nim-lang.org/). Note also that (from Fidget Readme): "Fidget aims to provide performant natively compiled cross platform UIs for any platform - Web with HTML5, Windows, macOS, Linux, iOS and Android with OpenGL."

Thank you for all your work and your contributions to opensource! I'm also looking forward to seeing the beta when it will be ready :)

**Comment by elhombretecla** [comment]
Hello everyone!

First of all thank you very much for your collaboration and insights. We will keep this in mind. It is a very interesting topic.

In the near future we will start working on Penpot integrations with external services. We will keep you informed when we get to that point in the project plan.

Thanks a lot!

**Comment by 12people** [comment]
Before this happens, a way to develop plug-ins for Penpot needs to be implemented. Here's the relevant suggestion thread for that: https://github.com/penpot/penpot/discussions/462
